### PR TITLE
IL-3018 get offered rate method in swap lens

### PR DESCRIPTION
**Pull request by kris-ipor**

### CODE REVIEW

File: contracts/amm/libraries/types/AmmInternalTypes.sol [hunk 1]
```diff
@@ -30,6 +30,14 @@ library AmmInternalTypes {
         IporTypes.AccruedIpor accruedIpor;
     }
 
+    struct RiskIndicatorsContext {
+        address asset;
+        address iporRiskManagementOracle;
+        IporTypes.SwapTenor tenor;
+        uint256 liquidityPool;
```
Comment (mario-ipor) [inline contracts/amm/libraries/types/AmmInternalTypes.sol:37]: change to liquidityPoolBalance

File: contracts/amm/AmmSwapsLens.sol [hunk 2]
```diff
@@ -139,6 +161,100 @@ contract AmmSwapsLens is IAmmSwapsLens {
...
+                )
+            ),
+            (uint256)
+        );
+    }
+
+    function _getRiskIndicators(
+        address asset,
+        IporTypes.SwapTenor tenor,
+        uint256 liquidityPool,
```
Comment (mario-ipor) [inline contracts/amm/AmmSwapsLens.sol:243]: Change to liquidityPoolBalance

File: contracts/amm/AmmSwapsLens.sol [hunk 3]
```diff
@@ -1,41 +1,53 @@
...
+
     constructor(
         SwapLensPoolConfiguration memory usdtCfg,
         SwapLensPoolConfiguration memory usdcCfg,
         SwapLensPoolConfiguration memory daiCfg,
         address iporOracle,
         address riskManagementOracle,
-        address router
+        address router,
+        address spreadRouter
```
Comment (mario-ipor) [inline contracts/amm/AmmSwapsLens.sol:50]: Align local deployment script also available in scripts folder

**Review by mario-ipor — CHANGES_REQUESTED** [review]

**Review by pete-ipor — APPROVED** [review]

File: contracts/amm/libraries/types/AmmInternalTypes.sol (hunk 1 above)
Comment (kris-ipor) [reply contracts/amm/libraries/types/AmmInternalTypes.sol]: done

File: contracts/amm/AmmSwapsLens.sol (hunk 2 above)
Comment (kris-ipor) [reply contracts/amm/AmmSwapsLens.sol]: done

File: contracts/amm/AmmSwapsLens.sol (hunk 3 above)
Comment (kris-ipor) [reply contracts/amm/AmmSwapsLens.sol:50]: done

**Pull request merged**